client/src/modules/journal/modals/search.modal.js

```diff
@@ -22,11 +27,38 @@ function JournalSearchModalController(Instance, Projects, Notify, Store, filters
   // assign already defined custom filters to searchQueries object
   vm.searchQueries = util.maskObjectFromKeys(filters, searchQueryOptions);
 
+  /**
+   * hasDefaultAccount is used to set a default account selection behavior
+   * if the search modal need to set account selection in default query panel we can send it
+   * as parameters
+   * @example
+   * <pre>
+   * Config.openSearchModal(filters, { hasDefaultAccount : true })
+   * </pre>
+   */
+  if (options.hasDefaultAccount) {
+    vm.hasDefaultAccount = true;
+  }
+
   // assign default filters
   if (filters.limit) {
     vm.defaultQueries.limit = filters.limit;
   }
 
+  // assing default account
+  if (filters.account_id) {
+    vm.defaultQueries.account_id = filters.account_id;
+  }
+
+  Account.read()
```

Comment:
Another way to reduce the data transfer is to do `Account.read(filters.account_id)` if it exists.  That way, we only load one account if we need it's `hrlabel` and otherwise we do not load any accounts at all.

I'll merge this and try and implement it to see if it works.